src/App.tsx

```diff
@@ -1,204 +1,105 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+
+type ErrorType =
+  | 'TodosLoad'
+  | 'EmptyTitle'
+  | 'UnableToAddTodo'
+  | 'UnableToDeleteTodo'
+  | 'UnableToUpdateTodo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [startingTodos, setStartingTodos] = useState<Todo[]>([]);
+  const [isTodoEditing, setIsTodoEditing] = useState(false);
+  const [selectedPostId, setSelectedPostId] = useState(0);
+  const [currentError, setCurrentError] = useState<ErrorType | ''>('');
+  const [selectedFilter, setSelectedFilter] = useState('#/');
```

Comment:
Selected filter value looks different. Also, you need to create a enum for  'all', 'active', 'completed' and use it everywhere